crates/bevy_ecs/src/schedule/stage.rs

```diff
@@ -167,6 +167,32 @@ impl SystemStage {
         self.add_system_to_set(system, 0)
     }
 
+    /// Topologically sorted parallel systems.
+    ///
+    /// Note that systems won't be fully-formed until the stage has been ran at least once.
```

Comment:
Nit: "run" is an "irregular verb" that behaves weirdly in cases like "has been". Every case of "ran" in these docs should be replaced with "run". English is a bad language :)

Reply:
I'm a native speaker and I still had to look this up to be sure my intuition wasn't wrong.

Reply:
Hmm looks like @Ratysz and I have differing opinions here (just saw the older threads here). Heres a source: https://ell.stackexchange.com/questions/133221/when-to-use-run-vs-when-to-use-ran

Reply:
Ah wait @Ratysz's comments are for a different case: "that want to be ran".  I don't know how to handle that one. My head hurts :)

Reply:
https://textranch.com/145625/should-be-run/or/should-be-ran/

Reply:
I think all cases should be "run" here.

Reply:
... wow. Clearly, we have to rewrite English in Rust. Apologies for adding to the confusion, @jakobhellermann!

I don't have links, I'm just going with my bilingual intuition - and it's throwing up its hands into the air right now.

Reply:
Done.